Lgo dev

LGO44 opened the pull request:
Based on the PlatformWiser 3.4.16 and the aioWiserHeatAPI 1.6.6
added equipment data
added some binary sensor
replaced a switch by a binary sensor

LGO44 committed:
add equipment data

Add equipment data for PTE, heatingactuators, smartplugs

LGO44 committed:
add binary sensors

for lights,  windowdoor,...

LGO44 committed:
Delete switch "Summer Discomfort Prevention"

replaced by a binary sensor

LGO44 committed:
V3.4.16.1

LGO44 committed:
error link for dev environment

LGO44 committed:
add extra attributes on sensors, add icon for shutter isopen and isclosed

LGO44 committed:
add switches

seasonal comfort enabled
enable Notification switch for devices (WiserDeviceEnableNotificationSwitch)
Over Power Notification Enabled
Under Power Notification Enabled

LGO44 committed:
add attributes on devsice signal

msp1974 commented:
Hey, sorry been busy on some other stuff for past few months. Let me look next week and try and get them in 3.4.17.

LGO44 committed:
add  Target lift for seasonnal comfort and summer comfort

LGO44 committed:
add MANUFACTURER_SCHNEIDER

LGO44 committed:
add select notification for binary-sensors , windowdoor select type

LGO44 committed:
Update for WindowDoor sensors

LGO44 committed:
Add sensors

Add weather sensor for seasonnal comfort

 Add pairing sensor to controll that the hub is paired to the Schneider account.

Add equipment sensor

LGO44 committed:
add switches to enable notification for devices or equipments or automations

LGO44 committed:
add  functions for equipments

LGO44 committed:
Version 3.4.17

LGO44 committed:
V3.4.17 PR582

LGO44 committed:
name WiserEquipmentPowerNotificationSwitch

LGO44 committed:
naming of sytem binary sensor and device_class

LGO44 committed:
fix  select led indicator for all lights

LGO44 committed:
bump to aioWiserHeatAPI v1.7.1

LGO44 commented:
rewrite in #631
